lib/rules/multiline-ternary.js

```diff
@@ -93,7 +116,16 @@ module.exports = {
                                 start: firstTokenOfTest.loc.start,
                                 end: lastTokenOfTest.loc.end
                             },
-                            messageId: "expectedTestCons"
+                            messageId: "expectedTestCons",
+                            fix: fixer => (hasComments ? null : (
+                                fixer.replaceTextRange(
+                                    [
+                                        lastTokenOfTest.range[1],
+                                        firstTokenOfConsequent.range[0]
+                                    ],
+                                    "\n? "
+                                )
+                            ))
```

Comment:
This could add an extra space after `?`, for example:

```js
a?b:c
```

will be autofixed to:

```js
a
? b
: c
```

Thoughts about just replacing range between `lastTokenOfTest` and `questionToken` with `"\n"`?

(same question for `messageId: "expectedConsAlt"`)

Reply:
seems it's the expected behavior as described in the original PR. But I agreed less changes is better.

I think we can leave the code as-is but adding/removing `"(\r?)\n"` - space issues could be fixed by rules like `no-trailing-spaces`, `space-infix-ops`. thoughts?

Reply:
I think no other core rule aside from `operator-linebreak` and `indent` can further change the fixed code if it wouldn't already change the original code. `space-infix-ops` only requires spaces, it can't be configured to disallow them.

Probably most people use exactly 1 space before and after `?`, so replacing the range between the nodes with `\n?<space>`, as it is done now, will work fine since the original code after `?` was the same `?<space>b` in that case.

A problem could be `?b` -> `?<space>b` if someone doesn't want a space after `?`, so they can complain that this change wasn't necessary.

Or something like `?<space><space>b` -> `?<space>b` or `?<tab>b` -> `?<space>b`, though all these are probably rare and it might be even good to remove whitespace that was possibly used for some alignments, so I don't have a strong opinion.